agent/containermetadata/metadata_manager.go

```diff
@@ -0,0 +1,151 @@
+// Copyright 2017 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
If I do not rename MetadataManager, is it still appropriate to rename this file? On one hand, there is no confusion with renaming as this file will be isolated in its own package. On the other this breaks the convention of all the other "_manager" files in the code base.